multiuser/machine-auth/che-multiuser-machine-authentication/src/main/java/org/eclipse/che/multiuser/machine/authentication/server/MachineTokenProviderImpl.java

```diff
@@ -33,8 +32,7 @@ public MachineTokenProviderImpl(MachineTokenRegistry tokenRegistry) {
   }
 
   @Override
-  public String getToken(String workspaceId) {
```

Comment:
I would rather left this method here and call getToken() with two params within it instead of adding 
`EnvironmentContext.getCurrent()...` in 15 other places

Reply:
yes, it makes sense. The reason why the method was removed is to force developers to explicitly call `EnvironmentContext.getCurrent()` in cases where they sure that user won't be anonymous, but I think it might be reworked in a better way, so I'll consider to back this method and adding a check(that user is not anonymous) before token retrieval.